agent/agent_worker.go

```diff
@@ -268,7 +268,10 @@ func (a *AgentWorker) Stop(graceful bool) {
 			// Kill the current job. Doesn't do anything if the job
 			// is already being killed, so it's safe to call
 			// multiple times.
-			a.jobRunner.CancelAndStop()
+			err := a.jobRunner.CancelAndStop()
```

Comment:
I will be forever disappointed that Go allows completely ignoring return vars. It's so incongruous with the overall helpful-strictness of the language.